images/win/scripts/Installers/Install-Haskell.ps1

```diff
@@ -35,5 +35,13 @@ Choco-Install -PackageName cabal
 Invoke-PesterTests -TestFile 'Haskell'
```

Comment:
@hasufell, We should run test after installation. Could you please move this statement at the end?

Reply:
@al-cheb done